src/sentry/monkey/__init__.py

```diff
@@ -14,38 +14,6 @@ def register_scheme(name):
 register_scheme("chrome-extension")
 
 
-def patch_httprequest_repr():
-    try:
-        from django.http import HttpRequest
-    except ImportError:
-        # This module is potentially imported before Django is installed
-        # during a setup.py run
-        return
-
-    # Intentionally strip all GET/POST/COOKIE values out of repr() for HttpRequest
-    # and subclass WSGIRequest. This prevents sensitive information from getting
-    # logged. This was yanked out of Django master anyhow.
-    # https://code.djangoproject.com/ticket/12098
-    def safe_httprequest_repr(self):
-        return f"<{self.__class__.__name__}: {self.method} {self.get_full_path()!r}>"
-
-    HttpRequest.__repr__ = safe_httprequest_repr  # type: ignore[method-assign]
-
-
-def patch_django_views_debug():
-    # Prevent exposing any Django SETTINGS on our debug error page
-    # This information is not useful for Sentry development
-    # and poses a significant security risk if this is exposed by accident
-    # in any production system if, by change, it were deployed
-    # with DEBUG=True.
-    try:
-        from django.views import debug
-    except ImportError:
-        return
-
-    debug.get_safe_settings = lambda: {}
```

Comment:
mypy helpfully pointed out this was doing nothing -- this hasn't done anything since django 1.x